pandora-server-directory/src/database/mongoDb.ts

```diff
@@ -325,7 +359,7 @@ export default class MongoDatabase implements PandoraDatabase {
 		return await this._chatrooms.find({
 			owners: { $elemMatch: { $in: [account] } },
 		})
-			.project<Pick<IChatRoomDirectoryData, 'id' | 'config' | 'owners'>>({ id: 1, config: 1, owners: 1 })
+			.project<Pick<IChatRoomDirectoryData, 'id' | 'config' | 'owners' | 'accessId'>>({ id: 1, config: 1, owners: 1, accessId: 1 })
```

Comment:
can we use `typeof CHATROOM_DIRECTORY_PROPERTIES[number]` here?

Reply:
I would agree with doing that if the pick filter object would also be autogenerated. I stuck with doing it manually as then TS catches that (for mongo the object can be any record, it doesn't check the exact property names, making it unsafe without being this explicit)